frontend/src/components/ItemPage.tsx

```diff
@@ -4,9 +4,14 @@ import {ItemModel} from "../model/ItemModel";
 import ItemNavigation from "./ItemNavigation";
 import ItemTable from "./ItemTable";
 import "../css/ItemPage.css"
+import Alert from "@mui/material/Alert";
 
 function ItemPage() {
     const [data, setData] = useState<ItemModel[]>([]);
+    const [failModal, setFailModal] = useState(false)
```

Comment:
name it errorModal, it's less confusing, because its also sucessModal with sucessMessage